package.json

```diff
@@ -16,19 +16,28 @@
   },
   "homepage": "https://github.com/OpenST/facilitator#readme",
   "dependencies": {
+    "commander": "2.20.0",
+    "fs-extra": "7.0.1",
+    "sqlite3": "4.0.8",
     "ts-node": "8.0.3",
     "typescript": "3.3.3333",
-    "web3": "1.0.0-beta.52",
-    "fs-extra": "7.0.1"
+    "web3": "1.0.0-beta.55",
+    "winston": "3.2.1"
   },
   "devDependencies": {
+    "@types/sqlite3": "3.1.5",
+    "@types/chai": "4.1.7",
+    "@types/fs-extra": "5.0.5",
```

Comment:
This is also in `dependencies` 🤔

Reply:
`fs-extra` package is under dependencies. Although `@types/fs-extra` is removed as it is not used.